example/zeto/README.md

```diff
@@ -2,35 +2,85 @@
 
 This example demonstrates a privacy-preserving fungible tokens using [Zeto](https://github.com/hyperledger-labs/zeto) on Paladin.
 
-See the [tutorial](https://lf-decentralized-trust-labs.github.io/paladin/head/tutorials/zeto/) for a detailed explanation.
+See the [tutorial](https://lf-decentralized-trust-labs.github.io/paladin/head/tutorials/zkp-cbdc/) for a detailed explanation.
 
 ## Pre-requisites
 
-Requires a local 3-node Paladin cluster running on `localhost:31548`, `localhost:31648`, and `localhost:31748`.
+### Local Paladin network
 
-## Run standalone
+This sample requires a local 3-node Paladin network running on `localhost:31548`, `localhost:31648`, and `localhost:31748`. You can get one setup by running the operator build:
```

Comment:
So are we now asserting that you cannot run the tutorials using a network configured as per the [Getting Started](https://lf-decentralized-trust-labs.github.io/paladin/head/getting-started/installation/) guide, but must build and deploy the operator from source?

That would be a significant pivot. We've seen many people want to play with the examples without taking on the burden of a whole source build.

Reply:
Oh I forgot about that, yes that should still be the primary way to get the local network setup. thanks for the feedback